x-pack/plugins/observability_ai_assistant/public/components/chat/chat_flyout.tsx

```diff
@@ -47,14 +50,14 @@ export function ChatFlyout({
     services: { http },
   } = useKibana();
 
-  const service = useObservabilityAIAssistant();
-
   const { euiTheme } = useEuiTheme();
 
   const router = useObservabilityAIAssistantRouter();
 
   const knowledgeBase = useKnowledgeBase();
 
+  useEffect(() => {}, [knowledgeBase.status.value?.ready]);
```

Comment:
What does this do?

Reply:
ah I need to re-create the chatService when this happens, will fix.

Reply:
I've removed the hook here [`a7ccdf1` (#38)](https://github.com/dgieselaar/kibana/pull/38/commits/a7ccdf1ad9b47d1fc1c1eb7c97b7c0b41f6da181), the refresh logic needs to happen in a separate place.